testsuite/classlibrary/TestIOUGens.sc

```diff
@@ -0,0 +1,127 @@
+TestIOUGens : UnitTest {
+    var server;
+
+	setUp {
+		server = Server(this.class.name);
+		server.bootSync;
+	}
+
+	tearDown {
+		server.quit;
+		server.remove;
+	}
+
+    //Test that InFeedback is equal to In when write / read are in the right order
+    test_InFeedback_equals_In {
+        var condition = Condition();
+        var bus = Bus.audio(server);
+
+        {
+            var writer = Out.ar(bus.index, SinOsc.ar(1));
+            var in = In.ar(bus.index);
+            var inFeedback = InFeedback.ar(bus.index);
+            inFeedback - in;
+        }.loadToFloatArray(0.5, server, { | data |
+            this.assertFloatEquals(data.sum, 0.0, "InFeedback is not equal to In", report:true);
```

Comment:
The report should be clearer, something like: "InFeedback should be equal to In when no other signal has been written to its bus"